src/main/java/htsjdk/samtools/util/GZIIndex.java

```diff
@@ -431,4 +432,34 @@ private static final ByteBuffer allocateBuffer(final int numberOfEntries,
         // creates a byte buffer in little-endian
         return ByteBuffer.allocate(size).order(ByteOrder.LITTLE_ENDIAN);
     }
+
+    /**
+     *
+     */
+    public static final class GZIIndexer implements Closeable {
+        private int uncompressedFileOffset = 0;
```

Comment:
unnecessary initializer
```suggestion
        private int uncompressedFileOffset;
```